frontend/src/components/camp/CollaboratorListItem.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <v-list-item class="px-0" two-line>
+    <v-list-item-avatar>
+      <v-img src="https://i.pravatar.cc/300" />
+    </v-list-item-avatar>
+    <v-list-item-content>
+      <v-list-item-title>{{ collaborator.user().username }}</v-list-item-title>
+      <v-list-item-subtitle>{{ collaborator.user().mail }}</v-list-item-subtitle>
+    </v-list-item-content>
+    <api-select
```

Comment:
Ich sehe hier kein kleines Dropdown-Pfeilchen, ist das so beabsichtigt? (Vielleicht war das auch vorher schon so)

Reply:
Das war vorher auch schon so. Würde ich in ein weiteres Issue aufnehmen.